monkey/monkey_island/cc/next_ui/.env

```
@@ -0,0 +1,17 @@
+#; Public variables
+
+NEXT_PUBLIC_PRODUCTION_KEY="production"
+NEXT_PUBLIC_TRUE_VALUE=True
+NEXT_PUBLIC_FALSE_VALUE=False
+NEXT_PUBLIC_EXTERNAL_API_BASE_URL="https://localhost:5000/api"
+NEXT_PUBLIC_API_ROUTES_URL="/api"
+NEXT_PUBLIC_DOMAIN="localhost"
+
+#; Private variables
+
+NEXTAUTH_SECRET="test"
+NEXTAUTH_URL="https://localhost:443"
+BASE_API="https://localhost:5000/api"
+
+#; Node variables
+NODE_TLS_REJECT_UNAUTHORIZED=0
```

This will ignore any errors from the certs. Have you tested the app without it?